### PR TITLE
Fix mobility setting visibility

### PR DESCRIPTION
**Pull request by vitalis0itu**
https://trello.com/c/jpajvjDs
and
https://trello.com/c/m3pVVAUm

also this was reported recently
https://trello.com/c/aAjvX5tm

### CODE REVIEW

File: src/components/SettingsComponent/SettingsComponent.js [hunk 1]
```diff
@@ -225,7 +229,10 @@ const SettingsComponent = ({ variant, classes }) => {
                       clickable
                       size="small"
                       label={uppercaseFirst(setting.title)}
-                      onDelete={() => handleOptionSelecting(setting.id, setting.category)}
+                      onDelete={() => {
+                        const settingId = setting.category === 'mobility' ? 'none' : setting.id;
+                        handleOptionSelecting(settingId, setting.category);
+                      }}
```
Comment (vitalis0itu) [inline src/components/SettingsComponent/SettingsComponent.js:235]: This fixes an issue where deleting a mobility setting does not work when settings component is collapsed.

File: src/utils/settings.js [hunk 2]
```diff
@@ -138,7 +138,7 @@ class SettingsUtility {
     }
     const data = [];
     const { mobility } = settings;
-    if (typeof mobility === 'string') {
+    if (typeof mobility === 'string' && mobility !== 'none') {
```
Comment (vitalis0itu) [inline src/utils/settings.js:141]: https://trello.com/c/m3pVVAUm

File: src/components/SettingsComponent/SettingsComponent.js (hunk 1 above)
Comment (vitalis0itu) [reply src/components/SettingsComponent/SettingsComponent.js:235]: https://trello.com/c/aAjvX5tm

**Review by mhieta — APPROVED** [review]
LGTM 👍 :shipit:

**Pull request merged**